plugins/modules/aci_cloud_cidr.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, nkatarmal-crest <nirav.katarmal@crestdatasys.com>
+# Copyright: (c) 2020, Cindy Zhao <cizhao@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: aci_cloud_cidr
+short_description: Manage CIDR under Cloud Context Profile (cloud:Cidr)
+description:
+-  Manage Cloud CIDR on Cisco Cloud ACI.
+notes:
+- More information about the internal APIC class B(cloud:Cidr) from
+  L(the APIC Management Information Model reference,https://developer.cisco.com/docs/apic-mim-ref/).
+- This module is only used to manage non_primary Cloud Cidr, the primary Cidr is created by aci_cloud_ctx_profile module.
+author:
+- Nirav (@nirav)
+- Cindy Zhao (@cizhao)
+options:
+  address:
+    description:
+    - CIDR ip and its submask.
+    type: str
+  description:
+    description:
+    - Description of the Cloud CIDR.
+    type: str
+  name_alias:
+    description:
+    - An alias for the name of the current object. This relates to the nameAlias field in ACI and is used to rename object without changing the DN.
+    type: str
+  tenant:
+    description:
+    - The name of the Tenant.
+    type: str
+  cloud_context_profile:
+    description:
+    - The name of the Cloud Context Profile.
+    type: str
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    choices: [ absent, present, query ]
+    default: present
+    type: str
+
```

Comment:
We had discussed to add a note to this module to explain why it is only non_primary. Not sure if we make it part of the description or a bottom note.

Reply:
can we have more than one note tag in the doc?
 I saw one note tag in line 17 so I just add the note under it.
should I add another bottom note for that?

Reply:
I don't know. One is probably more than enough.